### DISCUSSION TITLE
links/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#13] [General]
# links/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/links/

**Comment by TonyCrane** [comment]
欢迎来加友链捏 🥳
想加的可以在这里留个评论 🚀

**Reply by HK-SHAO** [reply]
感觉用MKdocs做类似博客的网站也很不醋

**Reply by timedegree** [reply]
好耶！

**Reply by timedegree** [reply]
_(:ɜ⌋∠)_加加我的

**Reply by LYK-love** [reply]
可以加我的吗? Link: lyk-love.cn

**Reply by LYK-love** [reply]
Name: LYK-love
Avatar: https://lyk-love.cn/images/white_flower1.jpg
Link: https://lyk-love.cn/
Description: NJU 19, UC Davis 23. Deep learning researcher.

**Reply by Auzers** [reply]
学长好！换个友链吧，我是24级zjuer
```
name：am 的 笔记本
link: https://Auzers.github.io/notes/
avatar: https://cdn.jsdelivr.net/gh/Auzers/drawingbed/wechat.jpeg
desc: 一个爱摆烂的人
```

**Comment by asandstar** [comment]
呜呜，请问您是20还是21的呢？从23春季朋辈小课堂来的~

**Reply by HobbitQia** [reply]
这事 21 的 kami😍

**Comment by wmh1024** [comment]
哇大佬太强了，可以换个友链吗~🤩

> * 我的名称： wmhwiki
> * 网站地址： https://wmhwiki.cn/
> * 描述： 鸟要高飞先振翅，人求上进先读书。
> * 头像： https://blog-1306153177.cos.ap-beijing.myqcloud.com/202308151203522.png

**Comment by w1ndys** [comment]
偶然发现大佬的博客，大佬好强啊，会开发Mkdocs插件，而且还打CTF（我是垃圾re手），赏脸换个友链不
```
- name: W1ndysの小屋
  link: https://blog.w1ndys.top/
  avatar: https://blog.w1ndys.top/img/about/avatar.png
  descr: 欲戴皇冠，必承其重
```

很喜欢你的Mkdocs风格，值得学习
